bakery/src/pipelines/distribution.js

```diff
@@ -16,10 +16,11 @@ const pipeline = (env) => {
 
   const resources = [
     {
-      name: 'cnx-recipes',
-      type: 'git',
+      name: 'cnx-recipes-output',
+      type: 'docker-image',
       source: {
-        uri: 'https://github.com/openstax/cnx-recipes.git'
+        repository: 'openstax/cnx-recipes-output',
+        tag: env.IMAGE_TAG || 'latest'
```

Comment:
I noticed there's not currently a `latest` tag of this image pushed up yet.

Reply:
Wow, good catch.

Reply:
I pushed a latest image for cnx-recipes-output - just a copy of `20200514.135335` for now